src/java/apiview-java-processor/src/main/java/com/azure/tools/apiview/processor/analysers/util/ASTUtils.java

```diff
@@ -209,7 +214,36 @@ public static String makeId(EnumConstantDeclaration enumDeclaration) {
     }
 
     public static String makeId(String fullPath) {
-        return MAKE_ID.matcher(fullPath).replaceAll("-");
+        if (fullPath == null || fullPath.isEmpty()) {
```

Comment:
what motivated this change?

Reply:
Hot path with a regex that can be replaced with a simple character search